python/plugins/processing/tests/testdata/qgis_algorithm_tests4.yaml

```diff
@@ -364,8 +364,11 @@ tests:
         type: vector
         pk: id_a
         compare:
+          unordered: true
           fields:
             fid: skip
+          geometry:
```

Comment:
why are these changes required?

Reply:
The test was failing because in the output polygon, the start/end vertex was not the same. Don't know why this happened, however we can consider the output is correct no matter which vertex in the polygon ring is first / last.